applications/GeoMechanicsApplication/custom_elements/transient_thermal_element.cpp

```diff
@@ -0,0 +1,854 @@
+    // ============================================================================================
+    template<unsigned int TDim, unsigned int TNumNodes>
+    void TransientThermalElement<TDim, TNumNodes>::EquationIdVector(
+        EquationIdVectorType& rResult,
+        const ProcessInfo& rCurrentProcessInfo) const
+    {
+        KRATOS_TRY
+    	//
+        const GeometryType& rGeom = this->GetGeometry();
+        const unsigned int N_DOF = this->GetNumberOfDOF();
+        unsigned int index = 0;
+        //
+        if (rResult.size() != N_DOF)
+            rResult.resize(N_DOF, false);
+        //
+        for (unsigned int i = 0; i < TNumNodes; ++i)
+        {
+            rResult[index++] = rGeom[i].GetDof(TEMPERATURE).EquationId();
+        }
+        //
+        KRATOS_CATCH("")
+    }
+
+	// ============================================================================================
+    // ============================================================================================
+    template<unsigned int TDim, unsigned int TNumNodes>
+    void TransientThermalElement<TDim, TNumNodes>::CalculateMassMatrix(
+        MatrixType& rMassMatrix,
+        const ProcessInfo& rCurrentProcessInfo)
+    {
+        KRATOS_TRY
+        //
+    	const unsigned int N_DOF = this->GetNumberOfDOF();
+        //
+        //Resizing mass matrix
+        if (rMassMatrix.size1() != N_DOF)
+            rMassMatrix.resize(N_DOF, N_DOF, false);
+        noalias(rMassMatrix) = ZeroMatrix(N_DOF, N_DOF);
+        //
+        KRATOS_CATCH("")
+    }
+
+    // ============================================================================================
+    // ============================================================================================
+    template<unsigned int TDim, unsigned int TNumNodes>
+    void TransientThermalElement<TDim, TNumNodes>::CalculateDampingMatrix(
+        MatrixType& rDampingMatrix,
+        const ProcessInfo& rCurrentProcessInfo)
+    {
+        KRATOS_TRY
+        //
+    	// Rayleigh Method (Damping Matrix = alpha*M + beta*K)
+        //
+    	const unsigned int N_DOF = this->GetNumberOfDOF();
+        //
+        // Compute Damping Matrix
+        if (rDampingMatrix.size1() != N_DOF)
+            rDampingMatrix.resize(N_DOF, N_DOF, false);
+        noalias(rDampingMatrix) = ZeroMatrix(N_DOF, N_DOF);
+        //
+        KRATOS_CATCH("")
+    }
+
+    // ============================================================================================
+    // ============================================================================================
+    template<unsigned int TDim, unsigned int TNumNodes>
+    void TransientThermalElement<TDim, TNumNodes>::GetValuesVector(
+        Vector& rValues,
+        int Step) const
+    {
+        KRATOS_TRY
+        //
+    	const unsigned int N_DOF = this->GetNumberOfDOF();
+        //
```

Comment:
this `//` is not honestly Kratonic style :)

Reply:
// are removed